modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-aws.adoc

```diff
@@ -8,7 +8,7 @@ To create a Redpanda cluster in your virtual private cloud (VPC), log in to http
 == Create a BYOC cluster
 
 . On the Clusters page, click *Create cluster*, then click *Create BYOC cluster*.
-. Enter a cluster name, then select the namespace, provider (AWS), xref:deploy:deployment-option/cloud/regions.adoc[region], availability zone type (single AZ or multi AZ) and zones, xref:deploy:deployment-option/cloud/cloud-overview.adoc#cluster-tiers[tier], and Redpanda version.
+. Enter a cluster name, then select the namespace, provider (AWS), xref:deploy:deployment-option/cloud/byoc-tiers.adoc[region, availability zone type (single AZ or multi AZ), zones], and Redpanda version.
```

Comment:
Just checking...so you specify an AZ type, a region, and a zone, separately?

Reply:
Yes here's the UI:
![image](https://github.com/redpanda-data/docs/assets/107008679/642cb57b-9018-4d7d-8a9a-0585224f85ff)